[2024-07-23 15:11:53,322] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: ETL_task.crawl_data 2024-07-23T15:11:51.997935+00:00 [queued]>
[2024-07-23 15:11:53,337] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: ETL_task.crawl_data 2024-07-23T15:11:51.997935+00:00 [queued]>
[2024-07-23 15:11:53,338] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-07-23 15:11:53,338] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-07-23 15:11:53,338] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-07-23 15:11:53,347] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): crawl_data> on 2024-07-23T15:11:51.997935+00:00
[2024-07-23 15:11:53,350] {standard_task_runner.py:52} INFO - Started process 1814 to run task
[2024-07-23 15:11:53,355] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ETL_task', 'crawl_data', '2024-07-23T15:11:51.997935+00:00', '--job-id', '325', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpbmfj4jjz', '--error-file', '/tmp/tmpob87kjr9']
[2024-07-23 15:11:53,358] {standard_task_runner.py:77} INFO - Job 325: Subtask crawl_data
[2024-07-23 15:11:53,396] {logging_mixin.py:104} INFO - Running <TaskInstance: ETL_task.crawl_data 2024-07-23T15:11:51.997935+00:00 [running]> on host 1026daea27fc
[2024-07-23 15:11:53,437] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=trihx2003@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_task
AIRFLOW_CTX_TASK_ID=crawl_data
AIRFLOW_CTX_EXECUTION_DATE=2024-07-23T15:11:51.997935+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-07-23T15:11:51.997935+00:00
[2024-07-23 15:11:53,662] {logging_mixin.py:104} INFO - Request failed with status code 429
[2024-07-23 15:11:53,663] {python.py:151} INFO - Done. Returned value was: None
[2024-07-23 15:11:53,672] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=ETL_task, task_id=crawl_data, execution_date=20240723T151151, start_date=20240723T151153, end_date=20240723T151153
[2024-07-23 15:11:53,697] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-23 15:11:53,730] {local_task_job.py:149} INFO - Task exited with return code 0
